sdk/storage/storage-blob/README.md

```diff
@@ -193,36 +194,32 @@ async function main() {
     `Upload block blob ${blobName} successfully`,
     uploadBlobResponse.requestId
   );
+```
 
-  // List blobs
-  marker = undefined;
-  do {
-    const listBlobsResponse = await containerClient.listBlobFlatSegment(
-      marker
-    );
+### List blobs inside a container
 
-    marker = listBlobsResponse.nextMarker;
-    for (const blob of listBlobsResponse.segment.blobItems) {
-      console.log(`Blob: ${blob.name}`);
-    }
-  } while (marker);
+Similar to listing containers.
```

Comment:
Can also add a link to the iterators sample for blobs ?

Reply:
adding

---
In reply to: [295983023](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295983023) [](ancestors = 295983023)